submissions/scripts/create_submissions.py

```diff
@@ -118,8 +118,9 @@ def parse_metadata(metadata_file: str, id3c_metadata_file: str = None) -> pd.Dat
         id3c_metadata['sequence_reason'] = id3c_metadata['sequence_reason'].fillna(value='Sentinel surveillance')
         id3c_metadata['originating_lab'] = 'Seattle Flu Study'
         id3c_metadata['submission_group'] = 'sfs'
-        # Label SCAN samples separately since they have different authors than SFS samples
-        id3c_metadata.loc[id3c_metadata['source'] == 'SCAN', 'submission_group'] = 'scan'
+        # Label SCAN and Cascadia samples separately since they have different authors than SFS samples
+        id3c_metadata.loc[id3c_metadata['source'].str.lower().str.strip() == 'scan', 'submission_group'] = 'scan'
+        id3c_metadata.loc[id3c_metadata['source'].str.lower().str.strip() == 'cascadia', 'submission_group'] = 'cascadia'
```

Comment:
surprised this is coming through from id3c with extraneous whitespace in strings. Not bad to handle here, but is there a view we should update upstream as well?

Reply:
There is no extra whitespace coming from ID3C, so probably not necessary, I just wanted to make it space and case-insensitive to be more robust.

Reply:
The Cascadia metadata was manually appended to the file for the most recent run, and 'cascadia` was all lowercase so this partly so there wasn't a mix of upper and lowercase being used. But ultimately that data will be coming from LIMS API, so also can probably drop this once that's in place.